anaconda.spec.in

```diff
@@ -305,7 +305,16 @@ update-desktop-database &> /dev/null || :
 %{_bindir}/anaconda-disable-nm-ibft-plugin
 %{_sbindir}/anaconda
 %{_sbindir}/handle-sshpw
-%{_datadir}/anaconda
+%dir %{_datadir}/anaconda
+%dir %{_datadir}/anaconda/addons
+%{_datadir}/anaconda/ui/
```

Comment:
Is this correct? Wouldn't this include whole `tui` and `gui` into the core package?